syntax="proto3";
package coord;
option go_package = "./;coord";

message cluster_handshake_request {
    string  NodeName = 1;
    string  Host = 2;
    int32   Port = 3;
    int64   Version = 4;
}

message cluster_handshake_response {
    int32   Code = 1;
}

message cluster_handshake_ack {
 
}

message gate_cluster_instead_request {
    uint64 userId = 1;
}

message gate_cluster_instead_response {
    uint64 userId = 1;
}

message gate_test_request {
    
}

message gate_test_response {
    
}

message base_handshake_request {
    int64   Version = 1;
}

message base_handshake_response {
    int32   Code = 1;
}

message base_handshake_ack {
 
}


